firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHttpClient.java

```diff
@@ -400,6 +401,16 @@ private static ConfigContainer extractConfigs(JSONObject fetchResponse, Date fet
         containerBuilder.withAbtExperiments(experimentDescriptions);
       }
 
+      JSONObject personalizationMetadata = null;
+      try {
+        personalizationMetadata = fetchResponse.getJSONObject(PERSONALIZATION_METADATA);
+      } catch (JSONException e) {
+        // Do nothing if personalizationMetadata does not exist.
```

Comment:
I'd recommend logging something anywhere we suppress an error. Silent failures limit the details customers can provide in support requests.

Reply:
Done.

Reply:
Thx

Thinking: `getJSONObject` throws if the key doesn't exist, which will be true for anyone not using P13n. We could check for the key first w `has`, but since this code only runs during fetch, I don't think it'll be spammy enough to justify that.

Reply:
Some other thoughts: we follow this `getJSONObject` pattern for other field above but don't log if there's nothing to get. RC had bug reports of spammy logging when we were logging warnings in the expected case for performance monitoring, so using at least the `INFO` level seems appropriate. I might even use `DEBUG` here to make it easier to hide this log message for users not using p13n.

Reply:
In general, if there's an error and it's actionable, I think we'd be wise to log a warning. In this case, it sounds like the common case will be to throw, so it's borderline not an error, and only actionable for folks using P13n, so it should not be `WARN` level. I guess in this case, we can't differentiate between an error case of missing metadata and the common case of not using the feature. `DEBUG` probably makes sense since it's a suppressed error, which folks would be looking for when debugging.